app/assets/stylesheets/components/_toolbar-dropdown.scss

```diff
@@ -7,6 +7,16 @@ $app-dropdown-arrow-spacing: $app-dropdown-arrow-size / 2;
   min-width: 120px;
   background: inherit;
   text-align: left;
+
+  .app-c-toolbar-dropdown__container {
```

Comment:
Perhaps this is related to the potential bug I've asked about in the polyfill method: would these styles conflict with the browser behaviour if the details element is actually supported?

Reply:
They shouldn't conflict, but again happy to extract this into its own piece of work.